.editorconfig

```diff
@@ -0,0 +1,16 @@
+# @see http://editorconfig.org/
+
+# This is the top-most .editorconfig file; do not search in parent directories.
+root = true
+
+# All files.
+[*]
+end_of_line = LF
+indent_style = space
+indent_size = 4
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[composer.{json,lock}]
+indent_size = 4
```

Comment:
Is this an override? Do we need this since it matches the indent_size above?

Reply:
Ah good call. Copy paste got me. (I stole this from Drupal).